paper/user_study.tex

```diff
@@ -49,8 +49,128 @@ \subsection{Results}
+        \cline{2-3}
+        & 4. The Schema editor GUI panel contains extensive metadata, but the code editor panel remains empty. & 4. bla bla bla \\
+        \cline{2-3}
+        & 5.To maintain consistency and readability, it should be remove the use of different font styles of property (bold, purple colour, * with purple colour and so on)  in the file editor within the GUI panel. & 5. bla bla bla \\
+        \cline{2-3}
+        & 6. The cursor styling in the file editor within the GUI panel should be improved to ensure it doesn't resemble a link cursor. & 6. Cursor styling is chnaged. \\
+        \cline{2-3}
+        & 7. The cross symbol has been removed from the dropdown button for enum properties in the GUI panel as it is deemed unnecessary. & 7. The cross symbol has been removed from the dropdown button for enum properties in the GUI panel. \\
+        \cline{2-3}
+        & 8. If the type of a property is as "any" in the Schema Editor, it should not default to being interpreted as the "string" type. & 8. bla bla \\
+        \cline{2-3}
+        & 9. Validation error should not showed as warning symbol, proper indication needed for validation error rather than warning symbol. & 9.The warning symbol has been removed, and a red cross indication has been implemented to clearly indicate validation errors in the user interface. \\
+        \cline{2-3}
+        & 10. Validation error should not showed as warning symbol, proper indication needed for validation error rather than warning symbol. & 10.The warning symbol has been removed, and a red cross indication has been implemented to clearly indicate validation errors in the user interface. \\
+        \cline{2-3}
+        & 11. After performing an undo or redo action, the cursor should jump to the appropriate location to reflect the changes made by the user.& 10. Will be consider for future work. \\
+        \hline
+    \end{tabular}
+    \caption* {User Study Feedback and Resolution (Continued)}
+\end{table*}
+
+
+\clearpage % Force a page break
+
+\begin{table*}
+
+    \centering
+    \small % Reduce font size
+    \setlength{\extrarowheight}{5pt} % Add extra vertical space
+    \renewcommand{\arraystretch}{1.5} % Adjust the vertical spacing between rows
+    \begin{tabular}{|c|p{0.4\linewidth}|p{0.4\linewidth}|}
+        \hline
+        User Study 2 & 1. A graph-based approach would be more intuitive for handling complex settings. & 1. Will be considered for future work. For now, we have a tree-based approach rather than a graph-based one. \\
+        \cline{2-3}
+        & 2. Providing immediate feedback to users when they enter incorrect ranges is essential to prevent them from inputting invalid values into the property. & 2. Will be consider for future work. For now, a red underline serves as an indicator to direct the user's attention when something is wrong. Additionally, when hovering over it, a dialog box can be used to inform the user of specific conditions. \\
```

Comment:
```suggestion
        & 2. Providing immediate feedback to users when they enter incorrect ranges is essential to prevent them from inputting invalid values into the property. & 2 For now, a red underline serves as an indicator to direct the user's attention when something is wrong. Additionally, when hovering over it, an overlay shows the exact errors that the user made. \\
```